### PR TITLE
Compute validator power & rewards for each checkpoint

### PR DESCRIPTION
**Pull request by itzmeanjan**
## Changes

- For each checkpoint, power & reward of that checkpoint signers, to be computed & stored
- This can be used as historical entry for each checkpoint sent from L2 to L1

### CODE REVIEW

File: root/src/mappings/checkpoint.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { Address, BigDecimal, BigInt, Value } from '@graphprotocol/graph-ts'
+import { Address, BigDecimal, BigInt, Bytes, Value } from '@graphprotocol/graph-ts'
```
Comment (jdkanani) [inline root/src/mappings/checkpoint.ts]: Can we fix lint errors?

**Review by jdkanani — CHANGES_REQUESTED** [review]

**Review by jdkanani — APPROVED** [review]

**Comment by jdkanani** [comment]
Reward calculation has a max cap on the checkpoint. Can we also take it into consideration? cc @imyourm8

**Review by Meshugah — APPROVED** [review]

File: root/src/mappings/checkpoint.ts (hunk 1 above)
Comment (itzmeanjan) [reply root/src/mappings/checkpoint.ts]: Done

**Review by itzmeanjan — COMMENTED** [review]
Done

**Comment by itzmeanjan** [comment]
We're reverting back to previous state. Please read more in  #26

**Comment by itzmeanjan** [comment]
Closing this PR, not required.

**Pull request closed without merging**